src/encoded/docs/schema-changes.md

```diff
@@ -154,44 +154,115 @@ This new object is an array of example objects that can successfully POST agains
+4. Add test in the **tests** directory to make sure your schema change functions as expected. For example, if we included a minor change in treatment object such that *μg/kg* could be specified as treatment units, the following test should allow one to test whether the update has been successfully implemented or not:
+
+        def test_treatment_patch_amount_units(testapp, treatment):
+            testapp.patch_json(
+            treatment['@id'],
+            {
+                'treatment_type': 'injection',
+                'amount': 20,
+                'amount_units': 'μg/kg'
+            },
+            status=200, 
+            ## Status 200 means the object was a successfully patched and the schema update works as expected.
+        )
 
-3. Update sample data, **data/inserts** directory, to test the changes made to the schema in **tests** directory. 
+5. Document the changes to the corresponding log file within the **schemas/changelogs** directory. For example, a minor change in the treatment object after version 11 that allowed one to use *μg/kg* as treatment units is shown below:
 
+### Minor changes since schema version 11
+
+* *μg/kg* can now be specified as amount units.
 
 ### Update schema version
+* Schema version has to be updated (bumped up by 1) if the change that is being introduced will lead to a potential invalidation of existing objects in the database. 
+
+* Examples include:
+    1) Changing the name of a property in the existing schema.
+    2) Removing a property from the existing schema.
+    3) A property that previously allowed free text is now changed to a possible list of allowed enums.
+    4) If an existing enum is removed from a property.
+    5) If one object is migrated into another object.
+    6) Addition of multiple new schema properties leading to substantial changes to an existing schema.
+
+* Most of the cases described above are examples where there will be a potential conflict for all the existing objects to be validated under the new schema. Hence, an additional step of adding an upgrader script will be required. This will ensure that all the existing objects will be changed such that they can fit into the new schema that is currently being implemented.
+
+* While the addition of multiple new schema properties is not going to invalidate any existing objects, it makes sense to update the schema version. This will be especially helpful to all the users and submitters who are trying to use these properties to either submit their data or while querying the database using scripts.
+
+**Follow the steps as outlined below**
+
+1. In the **schemas** directory, edit the existing properties in the corresponding JSON file named after the object and increment the schema version. One of the possible enums for for the genetic modifications object was "validation" up until schema version 6. As a part of schema version upate to 7, "validation" was removed and instead the term "characterization" was added.
```

Comment:
clearly separate the example from the instructions in step 1. It is easy to think that the GM example is part of the step 1.

Reply:
"validation" was an enum for "purpose" in genetic modification. Please fix your sentences.